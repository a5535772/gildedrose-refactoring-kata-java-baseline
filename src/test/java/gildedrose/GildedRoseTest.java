package gildedrose;

import org.approvaltests.Approvals;
import org.approvaltests.combinations.CombinationApprovals;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class GildedRoseTest {

    @Test
    public void updateQuality() {
        String name = "foo";
        int sellIn = 0;
        int quality = 0;
        String itemString = doUpdateQuality(name, sellIn, quality);

        // Approvals.verify(itemString);
        CombinationApprovals.verifyAllCombinations(
                this::doUpdateQuality,
                new String[]{name},
                new Integer[]{sellIn},
                new Integer[]{quality}
        );
    }

    private String doUpdateQuality(String name, int sellIn, int quality) {
        Item[] items = new Item[]{new Item(name, sellIn, quality)};
        GildedRose app = new GildedRose(items);
        app.updateQuality();
        // assertEquals("foo", app.items[0].name);
        return app.items[0].toString();
    }

}
