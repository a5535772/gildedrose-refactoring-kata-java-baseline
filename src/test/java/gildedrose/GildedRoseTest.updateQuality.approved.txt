[foo, -1, 0] => foo, -2, 0 
[foo, -1, 1] => foo, -2, 0 
[foo, -1, 2] => foo, -2, 0 
[foo, -1, 3] => foo, -2, 1 
[foo, -1, 46] => foo, -2, 44 
[foo, -1, 47] => foo, -2, 45 
[foo, -1, 48] => foo, -2, 46 
[foo, -1, 49] => foo, -2, 47 
[foo, -1, 50] => foo, -2, 48 
[foo, 0, 0] => foo, -1, 0 
[foo, 0, 1] => foo, -1, 0 
[foo, 0, 2] => foo, -1, 0 
[foo, 0, 3] => foo, -1, 1 
[foo, 0, 46] => foo, -1, 44 
[foo, 0, 47] => foo, -1, 45 
[foo, 0, 48] => foo, -1, 46 
[foo, 0, 49] => foo, -1, 47 
[foo, 0, 50] => foo, -1, 48 
[foo, 1, 0] => foo, 0, 0 
[foo, 1, 1] => foo, 0, 0 
[foo, 1, 2] => foo, 0, 1 
[foo, 1, 3] => foo, 0, 2 
[foo, 1, 46] => foo, 0, 45 
[foo, 1, 47] => foo, 0, 46 
[foo, 1, 48] => foo, 0, 47 
[foo, 1, 49] => foo, 0, 48 
[foo, 1, 50] => foo, 0, 49 
[foo, 4, 0] => foo, 3, 0 
[foo, 4, 1] => foo, 3, 0 
[foo, 4, 2] => foo, 3, 1 
[foo, 4, 3] => foo, 3, 2 
[foo, 4, 46] => foo, 3, 45 
[foo, 4, 47] => foo, 3, 46 
[foo, 4, 48] => foo, 3, 47 
[foo, 4, 49] => foo, 3, 48 
[foo, 4, 50] => foo, 3, 49 
[foo, 5, 0] => foo, 4, 0 
[foo, 5, 1] => foo, 4, 0 
[foo, 5, 2] => foo, 4, 1 
[foo, 5, 3] => foo, 4, 2 
[foo, 5, 46] => foo, 4, 45 
[foo, 5, 47] => foo, 4, 46 
[foo, 5, 48] => foo, 4, 47 
[foo, 5, 49] => foo, 4, 48 
[foo, 5, 50] => foo, 4, 49 
[foo, 6, 0] => foo, 5, 0 
[foo, 6, 1] => foo, 5, 0 
[foo, 6, 2] => foo, 5, 1 
[foo, 6, 3] => foo, 5, 2 
[foo, 6, 46] => foo, 5, 45 
[foo, 6, 47] => foo, 5, 46 
[foo, 6, 48] => foo, 5, 47 
[foo, 6, 49] => foo, 5, 48 
[foo, 6, 50] => foo, 5, 49 
[foo, 9, 0] => foo, 8, 0 
[foo, 9, 1] => foo, 8, 0 
[foo, 9, 2] => foo, 8, 1 
[foo, 9, 3] => foo, 8, 2 
[foo, 9, 46] => foo, 8, 45 
[foo, 9, 47] => foo, 8, 46 
[foo, 9, 48] => foo, 8, 47 
[foo, 9, 49] => foo, 8, 48 
[foo, 9, 50] => foo, 8, 49 
[foo, 10, 0] => foo, 9, 0 
[foo, 10, 1] => foo, 9, 0 
[foo, 10, 2] => foo, 9, 1 
[foo, 10, 3] => foo, 9, 2 
[foo, 10, 46] => foo, 9, 45 
[foo, 10, 47] => foo, 9, 46 
[foo, 10, 48] => foo, 9, 47 
[foo, 10, 49] => foo, 9, 48 
[foo, 10, 50] => foo, 9, 49 
[foo, 11, 0] => foo, 10, 0 
[foo, 11, 1] => foo, 10, 0 
[foo, 11, 2] => foo, 10, 1 
[foo, 11, 3] => foo, 10, 2 
[foo, 11, 46] => foo, 10, 45 
[foo, 11, 47] => foo, 10, 46 
[foo, 11, 48] => foo, 10, 47 
[foo, 11, 49] => foo, 10, 48 
[foo, 11, 50] => foo, 10, 49 
[Aged Brie, -1, 0] => Aged Brie, -2, 2 
[Aged Brie, -1, 1] => Aged Brie, -2, 3 
[Aged Brie, -1, 2] => Aged Brie, -2, 4 
[Aged Brie, -1, 3] => Aged Brie, -2, 5 
[Aged Brie, -1, 46] => Aged Brie, -2, 48 
[Aged Brie, -1, 47] => Aged Brie, -2, 49 
[Aged Brie, -1, 48] => Aged Brie, -2, 50 
[Aged Brie, -1, 49] => Aged Brie, -2, 50 
[Aged Brie, -1, 50] => Aged Brie, -2, 50 
[Aged Brie, 0, 0] => Aged Brie, -1, 2 
[Aged Brie, 0, 1] => Aged Brie, -1, 3 
[Aged Brie, 0, 2] => Aged Brie, -1, 4 
[Aged Brie, 0, 3] => Aged Brie, -1, 5 
[Aged Brie, 0, 46] => Aged Brie, -1, 48 
[Aged Brie, 0, 47] => Aged Brie, -1, 49 
[Aged Brie, 0, 48] => Aged Brie, -1, 50 
[Aged Brie, 0, 49] => Aged Brie, -1, 50 
[Aged Brie, 0, 50] => Aged Brie, -1, 50 
[Aged Brie, 1, 0] => Aged Brie, 0, 1 
[Aged Brie, 1, 1] => Aged Brie, 0, 2 
[Aged Brie, 1, 2] => Aged Brie, 0, 3 
[Aged Brie, 1, 3] => Aged Brie, 0, 4 
[Aged Brie, 1, 46] => Aged Brie, 0, 47 
[Aged Brie, 1, 47] => Aged Brie, 0, 48 
[Aged Brie, 1, 48] => Aged Brie, 0, 49 
[Aged Brie, 1, 49] => Aged Brie, 0, 50 
[Aged Brie, 1, 50] => Aged Brie, 0, 50 
[Aged Brie, 4, 0] => Aged Brie, 3, 1 
[Aged Brie, 4, 1] => Aged Brie, 3, 2 
[Aged Brie, 4, 2] => Aged Brie, 3, 3 
[Aged Brie, 4, 3] => Aged Brie, 3, 4 
[Aged Brie, 4, 46] => Aged Brie, 3, 47 
[Aged Brie, 4, 47] => Aged Brie, 3, 48 
[Aged Brie, 4, 48] => Aged Brie, 3, 49 
[Aged Brie, 4, 49] => Aged Brie, 3, 50 
[Aged Brie, 4, 50] => Aged Brie, 3, 50 
[Aged Brie, 5, 0] => Aged Brie, 4, 1 
[Aged Brie, 5, 1] => Aged Brie, 4, 2 
[Aged Brie, 5, 2] => Aged Brie, 4, 3 
[Aged Brie, 5, 3] => Aged Brie, 4, 4 
[Aged Brie, 5, 46] => Aged Brie, 4, 47 
[Aged Brie, 5, 47] => Aged Brie, 4, 48 
[Aged Brie, 5, 48] => Aged Brie, 4, 49 
[Aged Brie, 5, 49] => Aged Brie, 4, 50 
[Aged Brie, 5, 50] => Aged Brie, 4, 50 
[Aged Brie, 6, 0] => Aged Brie, 5, 1 
[Aged Brie, 6, 1] => Aged Brie, 5, 2 
[Aged Brie, 6, 2] => Aged Brie, 5, 3 
[Aged Brie, 6, 3] => Aged Brie, 5, 4 
[Aged Brie, 6, 46] => Aged Brie, 5, 47 
[Aged Brie, 6, 47] => Aged Brie, 5, 48 
[Aged Brie, 6, 48] => Aged Brie, 5, 49 
[Aged Brie, 6, 49] => Aged Brie, 5, 50 
[Aged Brie, 6, 50] => Aged Brie, 5, 50 
[Aged Brie, 9, 0] => Aged Brie, 8, 1 
[Aged Brie, 9, 1] => Aged Brie, 8, 2 
[Aged Brie, 9, 2] => Aged Brie, 8, 3 
[Aged Brie, 9, 3] => Aged Brie, 8, 4 
[Aged Brie, 9, 46] => Aged Brie, 8, 47 
[Aged Brie, 9, 47] => Aged Brie, 8, 48 
[Aged Brie, 9, 48] => Aged Brie, 8, 49 
[Aged Brie, 9, 49] => Aged Brie, 8, 50 
[Aged Brie, 9, 50] => Aged Brie, 8, 50 
[Aged Brie, 10, 0] => Aged Brie, 9, 1 
[Aged Brie, 10, 1] => Aged Brie, 9, 2 
[Aged Brie, 10, 2] => Aged Brie, 9, 3 
[Aged Brie, 10, 3] => Aged Brie, 9, 4 
[Aged Brie, 10, 46] => Aged Brie, 9, 47 
[Aged Brie, 10, 47] => Aged Brie, 9, 48 
[Aged Brie, 10, 48] => Aged Brie, 9, 49 
[Aged Brie, 10, 49] => Aged Brie, 9, 50 
[Aged Brie, 10, 50] => Aged Brie, 9, 50 
[Aged Brie, 11, 0] => Aged Brie, 10, 1 
[Aged Brie, 11, 1] => Aged Brie, 10, 2 
[Aged Brie, 11, 2] => Aged Brie, 10, 3 
[Aged Brie, 11, 3] => Aged Brie, 10, 4 
[Aged Brie, 11, 46] => Aged Brie, 10, 47 
[Aged Brie, 11, 47] => Aged Brie, 10, 48 
[Aged Brie, 11, 48] => Aged Brie, 10, 49 
[Aged Brie, 11, 49] => Aged Brie, 10, 50 
[Aged Brie, 11, 50] => Aged Brie, 10, 50 
[Backstage passes to a TAFKAL80ETC concert, -1, 0] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 1] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 2] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 3] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 46] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 47] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 48] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 49] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 50] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 0] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 1] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 2] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 3] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 46] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 47] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 48] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 49] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 50] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 1, 0] => Backstage passes to a TAFKAL80ETC concert, 0, 3 
[Backstage passes to a TAFKAL80ETC concert, 1, 1] => Backstage passes to a TAFKAL80ETC concert, 0, 4 
[Backstage passes to a TAFKAL80ETC concert, 1, 2] => Backstage passes to a TAFKAL80ETC concert, 0, 5 
[Backstage passes to a TAFKAL80ETC concert, 1, 3] => Backstage passes to a TAFKAL80ETC concert, 0, 6 
[Backstage passes to a TAFKAL80ETC concert, 1, 46] => Backstage passes to a TAFKAL80ETC concert, 0, 49 
[Backstage passes to a TAFKAL80ETC concert, 1, 47] => Backstage passes to a TAFKAL80ETC concert, 0, 50 
[Backstage passes to a TAFKAL80ETC concert, 1, 48] => Backstage passes to a TAFKAL80ETC concert, 0, 50 
[Backstage passes to a TAFKAL80ETC concert, 1, 49] => Backstage passes to a TAFKAL80ETC concert, 0, 50 
[Backstage passes to a TAFKAL80ETC concert, 1, 50] => Backstage passes to a TAFKAL80ETC concert, 0, 50 
[Backstage passes to a TAFKAL80ETC concert, 4, 0] => Backstage passes to a TAFKAL80ETC concert, 3, 3 
[Backstage passes to a TAFKAL80ETC concert, 4, 1] => Backstage passes to a TAFKAL80ETC concert, 3, 4 
[Backstage passes to a TAFKAL80ETC concert, 4, 2] => Backstage passes to a TAFKAL80ETC concert, 3, 5 
[Backstage passes to a TAFKAL80ETC concert, 4, 3] => Backstage passes to a TAFKAL80ETC concert, 3, 6 
[Backstage passes to a TAFKAL80ETC concert, 4, 46] => Backstage passes to a TAFKAL80ETC concert, 3, 49 
[Backstage passes to a TAFKAL80ETC concert, 4, 47] => Backstage passes to a TAFKAL80ETC concert, 3, 50 
[Backstage passes to a TAFKAL80ETC concert, 4, 48] => Backstage passes to a TAFKAL80ETC concert, 3, 50 
[Backstage passes to a TAFKAL80ETC concert, 4, 49] => Backstage passes to a TAFKAL80ETC concert, 3, 50 
[Backstage passes to a TAFKAL80ETC concert, 4, 50] => Backstage passes to a TAFKAL80ETC concert, 3, 50 
[Backstage passes to a TAFKAL80ETC concert, 5, 0] => Backstage passes to a TAFKAL80ETC concert, 4, 3 
[Backstage passes to a TAFKAL80ETC concert, 5, 1] => Backstage passes to a TAFKAL80ETC concert, 4, 4 
[Backstage passes to a TAFKAL80ETC concert, 5, 2] => Backstage passes to a TAFKAL80ETC concert, 4, 5 
[Backstage passes to a TAFKAL80ETC concert, 5, 3] => Backstage passes to a TAFKAL80ETC concert, 4, 6 
[Backstage passes to a TAFKAL80ETC concert, 5, 46] => Backstage passes to a TAFKAL80ETC concert, 4, 49 
[Backstage passes to a TAFKAL80ETC concert, 5, 47] => Backstage passes to a TAFKAL80ETC concert, 4, 50 
[Backstage passes to a TAFKAL80ETC concert, 5, 48] => Backstage passes to a TAFKAL80ETC concert, 4, 50 
[Backstage passes to a TAFKAL80ETC concert, 5, 49] => Backstage passes to a TAFKAL80ETC concert, 4, 50 
[Backstage passes to a TAFKAL80ETC concert, 5, 50] => Backstage passes to a TAFKAL80ETC concert, 4, 50 
[Backstage passes to a TAFKAL80ETC concert, 6, 0] => Backstage passes to a TAFKAL80ETC concert, 5, 2 
[Backstage passes to a TAFKAL80ETC concert, 6, 1] => Backstage passes to a TAFKAL80ETC concert, 5, 3 
[Backstage passes to a TAFKAL80ETC concert, 6, 2] => Backstage passes to a TAFKAL80ETC concert, 5, 4 
[Backstage passes to a TAFKAL80ETC concert, 6, 3] => Backstage passes to a TAFKAL80ETC concert, 5, 5 
[Backstage passes to a TAFKAL80ETC concert, 6, 46] => Backstage passes to a TAFKAL80ETC concert, 5, 48 
[Backstage passes to a TAFKAL80ETC concert, 6, 47] => Backstage passes to a TAFKAL80ETC concert, 5, 49 
[Backstage passes to a TAFKAL80ETC concert, 6, 48] => Backstage passes to a TAFKAL80ETC concert, 5, 50 
[Backstage passes to a TAFKAL80ETC concert, 6, 49] => Backstage passes to a TAFKAL80ETC concert, 5, 50 
[Backstage passes to a TAFKAL80ETC concert, 6, 50] => Backstage passes to a TAFKAL80ETC concert, 5, 50 
[Backstage passes to a TAFKAL80ETC concert, 9, 0] => Backstage passes to a TAFKAL80ETC concert, 8, 2 
[Backstage passes to a TAFKAL80ETC concert, 9, 1] => Backstage passes to a TAFKAL80ETC concert, 8, 3 
[Backstage passes to a TAFKAL80ETC concert, 9, 2] => Backstage passes to a TAFKAL80ETC concert, 8, 4 
[Backstage passes to a TAFKAL80ETC concert, 9, 3] => Backstage passes to a TAFKAL80ETC concert, 8, 5 
[Backstage passes to a TAFKAL80ETC concert, 9, 46] => Backstage passes to a TAFKAL80ETC concert, 8, 48 
[Backstage passes to a TAFKAL80ETC concert, 9, 47] => Backstage passes to a TAFKAL80ETC concert, 8, 49 
[Backstage passes to a TAFKAL80ETC concert, 9, 48] => Backstage passes to a TAFKAL80ETC concert, 8, 50 
[Backstage passes to a TAFKAL80ETC concert, 9, 49] => Backstage passes to a TAFKAL80ETC concert, 8, 50 
[Backstage passes to a TAFKAL80ETC concert, 9, 50] => Backstage passes to a TAFKAL80ETC concert, 8, 50 
[Backstage passes to a TAFKAL80ETC concert, 10, 0] => Backstage passes to a TAFKAL80ETC concert, 9, 2 
[Backstage passes to a TAFKAL80ETC concert, 10, 1] => Backstage passes to a TAFKAL80ETC concert, 9, 3 
[Backstage passes to a TAFKAL80ETC concert, 10, 2] => Backstage passes to a TAFKAL80ETC concert, 9, 4 
[Backstage passes to a TAFKAL80ETC concert, 10, 3] => Backstage passes to a TAFKAL80ETC concert, 9, 5 
[Backstage passes to a TAFKAL80ETC concert, 10, 46] => Backstage passes to a TAFKAL80ETC concert, 9, 48 
[Backstage passes to a TAFKAL80ETC concert, 10, 47] => Backstage passes to a TAFKAL80ETC concert, 9, 49 
[Backstage passes to a TAFKAL80ETC concert, 10, 48] => Backstage passes to a TAFKAL80ETC concert, 9, 50 
[Backstage passes to a TAFKAL80ETC concert, 10, 49] => Backstage passes to a TAFKAL80ETC concert, 9, 50 
[Backstage passes to a TAFKAL80ETC concert, 10, 50] => Backstage passes to a TAFKAL80ETC concert, 9, 50 
[Backstage passes to a TAFKAL80ETC concert, 11, 0] => Backstage passes to a TAFKAL80ETC concert, 10, 1 
[Backstage passes to a TAFKAL80ETC concert, 11, 1] => Backstage passes to a TAFKAL80ETC concert, 10, 2 
[Backstage passes to a TAFKAL80ETC concert, 11, 2] => Backstage passes to a TAFKAL80ETC concert, 10, 3 
[Backstage passes to a TAFKAL80ETC concert, 11, 3] => Backstage passes to a TAFKAL80ETC concert, 10, 4 
[Backstage passes to a TAFKAL80ETC concert, 11, 46] => Backstage passes to a TAFKAL80ETC concert, 10, 47 
[Backstage passes to a TAFKAL80ETC concert, 11, 47] => Backstage passes to a TAFKAL80ETC concert, 10, 48 
[Backstage passes to a TAFKAL80ETC concert, 11, 48] => Backstage passes to a TAFKAL80ETC concert, 10, 49 
[Backstage passes to a TAFKAL80ETC concert, 11, 49] => Backstage passes to a TAFKAL80ETC concert, 10, 50 
[Backstage passes to a TAFKAL80ETC concert, 11, 50] => Backstage passes to a TAFKAL80ETC concert, 10, 50 
[Sulfuras, Hand of Ragnaros, -1, 0] => Sulfuras, Hand of Ragnaros, -1, 0 
[Sulfuras, Hand of Ragnaros, -1, 1] => Sulfuras, Hand of Ragnaros, -1, 1 
[Sulfuras, Hand of Ragnaros, -1, 2] => Sulfuras, Hand of Ragnaros, -1, 2 
[Sulfuras, Hand of Ragnaros, -1, 3] => Sulfuras, Hand of Ragnaros, -1, 3 
[Sulfuras, Hand of Ragnaros, -1, 46] => Sulfuras, Hand of Ragnaros, -1, 46 
[Sulfuras, Hand of Ragnaros, -1, 47] => Sulfuras, Hand of Ragnaros, -1, 47 
[Sulfuras, Hand of Ragnaros, -1, 48] => Sulfuras, Hand of Ragnaros, -1, 48 
[Sulfuras, Hand of Ragnaros, -1, 49] => Sulfuras, Hand of Ragnaros, -1, 49 
[Sulfuras, Hand of Ragnaros, -1, 50] => Sulfuras, Hand of Ragnaros, -1, 50 
[Sulfuras, Hand of Ragnaros, 0, 0] => Sulfuras, Hand of Ragnaros, 0, 0 
[Sulfuras, Hand of Ragnaros, 0, 1] => Sulfuras, Hand of Ragnaros, 0, 1 
[Sulfuras, Hand of Ragnaros, 0, 2] => Sulfuras, Hand of Ragnaros, 0, 2 
[Sulfuras, Hand of Ragnaros, 0, 3] => Sulfuras, Hand of Ragnaros, 0, 3 
[Sulfuras, Hand of Ragnaros, 0, 46] => Sulfuras, Hand of Ragnaros, 0, 46 
[Sulfuras, Hand of Ragnaros, 0, 47] => Sulfuras, Hand of Ragnaros, 0, 47 
[Sulfuras, Hand of Ragnaros, 0, 48] => Sulfuras, Hand of Ragnaros, 0, 48 
[Sulfuras, Hand of Ragnaros, 0, 49] => Sulfuras, Hand of Ragnaros, 0, 49 
[Sulfuras, Hand of Ragnaros, 0, 50] => Sulfuras, Hand of Ragnaros, 0, 50 
[Sulfuras, Hand of Ragnaros, 1, 0] => Sulfuras, Hand of Ragnaros, 1, 0 
[Sulfuras, Hand of Ragnaros, 1, 1] => Sulfuras, Hand of Ragnaros, 1, 1 
[Sulfuras, Hand of Ragnaros, 1, 2] => Sulfuras, Hand of Ragnaros, 1, 2 
[Sulfuras, Hand of Ragnaros, 1, 3] => Sulfuras, Hand of Ragnaros, 1, 3 
[Sulfuras, Hand of Ragnaros, 1, 46] => Sulfuras, Hand of Ragnaros, 1, 46 
[Sulfuras, Hand of Ragnaros, 1, 47] => Sulfuras, Hand of Ragnaros, 1, 47 
[Sulfuras, Hand of Ragnaros, 1, 48] => Sulfuras, Hand of Ragnaros, 1, 48 
[Sulfuras, Hand of Ragnaros, 1, 49] => Sulfuras, Hand of Ragnaros, 1, 49 
[Sulfuras, Hand of Ragnaros, 1, 50] => Sulfuras, Hand of Ragnaros, 1, 50 
[Sulfuras, Hand of Ragnaros, 4, 0] => Sulfuras, Hand of Ragnaros, 4, 0 
[Sulfuras, Hand of Ragnaros, 4, 1] => Sulfuras, Hand of Ragnaros, 4, 1 
[Sulfuras, Hand of Ragnaros, 4, 2] => Sulfuras, Hand of Ragnaros, 4, 2 
[Sulfuras, Hand of Ragnaros, 4, 3] => Sulfuras, Hand of Ragnaros, 4, 3 
[Sulfuras, Hand of Ragnaros, 4, 46] => Sulfuras, Hand of Ragnaros, 4, 46 
[Sulfuras, Hand of Ragnaros, 4, 47] => Sulfuras, Hand of Ragnaros, 4, 47 
[Sulfuras, Hand of Ragnaros, 4, 48] => Sulfuras, Hand of Ragnaros, 4, 48 
[Sulfuras, Hand of Ragnaros, 4, 49] => Sulfuras, Hand of Ragnaros, 4, 49 
[Sulfuras, Hand of Ragnaros, 4, 50] => Sulfuras, Hand of Ragnaros, 4, 50 
[Sulfuras, Hand of Ragnaros, 5, 0] => Sulfuras, Hand of Ragnaros, 5, 0 
[Sulfuras, Hand of Ragnaros, 5, 1] => Sulfuras, Hand of Ragnaros, 5, 1 
[Sulfuras, Hand of Ragnaros, 5, 2] => Sulfuras, Hand of Ragnaros, 5, 2 
[Sulfuras, Hand of Ragnaros, 5, 3] => Sulfuras, Hand of Ragnaros, 5, 3 
[Sulfuras, Hand of Ragnaros, 5, 46] => Sulfuras, Hand of Ragnaros, 5, 46 
[Sulfuras, Hand of Ragnaros, 5, 47] => Sulfuras, Hand of Ragnaros, 5, 47 
[Sulfuras, Hand of Ragnaros, 5, 48] => Sulfuras, Hand of Ragnaros, 5, 48 
[Sulfuras, Hand of Ragnaros, 5, 49] => Sulfuras, Hand of Ragnaros, 5, 49 
[Sulfuras, Hand of Ragnaros, 5, 50] => Sulfuras, Hand of Ragnaros, 5, 50 
[Sulfuras, Hand of Ragnaros, 6, 0] => Sulfuras, Hand of Ragnaros, 6, 0 
[Sulfuras, Hand of Ragnaros, 6, 1] => Sulfuras, Hand of Ragnaros, 6, 1 
[Sulfuras, Hand of Ragnaros, 6, 2] => Sulfuras, Hand of Ragnaros, 6, 2 
[Sulfuras, Hand of Ragnaros, 6, 3] => Sulfuras, Hand of Ragnaros, 6, 3 
[Sulfuras, Hand of Ragnaros, 6, 46] => Sulfuras, Hand of Ragnaros, 6, 46 
[Sulfuras, Hand of Ragnaros, 6, 47] => Sulfuras, Hand of Ragnaros, 6, 47 
[Sulfuras, Hand of Ragnaros, 6, 48] => Sulfuras, Hand of Ragnaros, 6, 48 
[Sulfuras, Hand of Ragnaros, 6, 49] => Sulfuras, Hand of Ragnaros, 6, 49 
[Sulfuras, Hand of Ragnaros, 6, 50] => Sulfuras, Hand of Ragnaros, 6, 50 
[Sulfuras, Hand of Ragnaros, 9, 0] => Sulfuras, Hand of Ragnaros, 9, 0 
[Sulfuras, Hand of Ragnaros, 9, 1] => Sulfuras, Hand of Ragnaros, 9, 1 
[Sulfuras, Hand of Ragnaros, 9, 2] => Sulfuras, Hand of Ragnaros, 9, 2 
[Sulfuras, Hand of Ragnaros, 9, 3] => Sulfuras, Hand of Ragnaros, 9, 3 
[Sulfuras, Hand of Ragnaros, 9, 46] => Sulfuras, Hand of Ragnaros, 9, 46 
[Sulfuras, Hand of Ragnaros, 9, 47] => Sulfuras, Hand of Ragnaros, 9, 47 
[Sulfuras, Hand of Ragnaros, 9, 48] => Sulfuras, Hand of Ragnaros, 9, 48 
[Sulfuras, Hand of Ragnaros, 9, 49] => Sulfuras, Hand of Ragnaros, 9, 49 
[Sulfuras, Hand of Ragnaros, 9, 50] => Sulfuras, Hand of Ragnaros, 9, 50 
[Sulfuras, Hand of Ragnaros, 10, 0] => Sulfuras, Hand of Ragnaros, 10, 0 
[Sulfuras, Hand of Ragnaros, 10, 1] => Sulfuras, Hand of Ragnaros, 10, 1 
[Sulfuras, Hand of Ragnaros, 10, 2] => Sulfuras, Hand of Ragnaros, 10, 2 
[Sulfuras, Hand of Ragnaros, 10, 3] => Sulfuras, Hand of Ragnaros, 10, 3 
[Sulfuras, Hand of Ragnaros, 10, 46] => Sulfuras, Hand of Ragnaros, 10, 46 
[Sulfuras, Hand of Ragnaros, 10, 47] => Sulfuras, Hand of Ragnaros, 10, 47 
[Sulfuras, Hand of Ragnaros, 10, 48] => Sulfuras, Hand of Ragnaros, 10, 48 
[Sulfuras, Hand of Ragnaros, 10, 49] => Sulfuras, Hand of Ragnaros, 10, 49 
[Sulfuras, Hand of Ragnaros, 10, 50] => Sulfuras, Hand of Ragnaros, 10, 50 
[Sulfuras, Hand of Ragnaros, 11, 0] => Sulfuras, Hand of Ragnaros, 11, 0 
[Sulfuras, Hand of Ragnaros, 11, 1] => Sulfuras, Hand of Ragnaros, 11, 1 
[Sulfuras, Hand of Ragnaros, 11, 2] => Sulfuras, Hand of Ragnaros, 11, 2 
[Sulfuras, Hand of Ragnaros, 11, 3] => Sulfuras, Hand of Ragnaros, 11, 3 
[Sulfuras, Hand of Ragnaros, 11, 46] => Sulfuras, Hand of Ragnaros, 11, 46 
[Sulfuras, Hand of Ragnaros, 11, 47] => Sulfuras, Hand of Ragnaros, 11, 47 
[Sulfuras, Hand of Ragnaros, 11, 48] => Sulfuras, Hand of Ragnaros, 11, 48 
[Sulfuras, Hand of Ragnaros, 11, 49] => Sulfuras, Hand of Ragnaros, 11, 49 
[Sulfuras, Hand of Ragnaros, 11, 50] => Sulfuras, Hand of Ragnaros, 11, 50 
